### DISCUSSION TITLE
Where can I find .pbiviz file?

### DISCUSSION THREAD

**Discussion by peegee96** [microsoft/SandDance#588] [Q&A] [answered]
Hi I am not a programmer, however I wanted to use the sanddance version (in Power BI) with solution for inverted z-axis (https://github.com/microsoft/SandDance/issues/546). Where can i find .PBIVIZ file in the resources or is there an easy way to create the this PBIVIZ file?
The "Get Visual" in PowerBI still refers to the old version 4.0.0.0 with inverted Z-axis.

**Comment by danmarshall** [comment]
Hello, you can find links to downloads on our [examples page](https://microsoft.github.io/SandDance/examples/). You should see a link for a 4.1.0 .pbiviz file.

**Reply by peegee96** [reply]
Thanks Dan, I did download the 4.1.0 in PowerBI, but it looks like the inverted Z-axis issues is not resolved, and the snapshot (camera icon) is also missing. I've uploaded the same data to the web app (https://microsoft.github.io/SandDance/tests/v4/es6/app) and the graphs appear fine and it also has the snapshot button.

![Sanddance_4_1_0](https://user-images.githubusercontent.com/122052410/213274555-847836d4-f36f-4d20-88d5-826d08f1272a.png)

**Reply by danmarshall** [reply]
Thanks @peegee96 - I will provide an updated .pbiviz file shortly with the z-axis fix. The snapshots feature is however deprecated in the Power BI visual, in favor of using PBI's native bookmarks instead - see #589 . If you have a legacy report which already contains snapshots, the camera icon will appear and be functional.

**Reply by peegee96** [reply]
Thanks @danmarshall, I really appreciate your quick reply and help.

**Comment by danmarshall** [comment]
https://microsoft.github.io/SandDance/dist/powerbi/v4/SandDance201929976D117A654D0BAB8E96507442D80B.4.1.1.pbiviz

**Comment by peegee96** [comment, marked as answer]
Thanks Dan, I downloaded the 4.1.1 and the axis is still inverted. I think there might be something wrong with my PBI. 
I am using PBI (version May 2022) and I am loading the custom visuals from file, however when I check the version it still show 4.0.0 even though I am loading the correct (4.1.1) file. I even did the entire process from scratch with internet turned off (just in case) but it keeps showing v4.0.0. 
I will report back on if the solution worked or not once I figure out my PBI issue.
![Untitled](https://user-images.githubusercontent.com/122052410/213541042-e59d5a60-8a4b-4755-b6f7-6bf76df647cf.png)

**Reply by danmarshall** [reply]
Oh, right. This is for your protection :) You can get around this by going into File > Options and Settings > Options > Current File > Report Settings > Developer Mode, then importing the visual from a file (again) and it should show a dialog asking you to confirm that you want to override with the new version.

**Reply by peegee96** [reply]
This worked. Thank you very much for your help.
